frontend/src/metabase/browse/components/CollectionBreadcrumbsWithTooltip.styled.tsx

```diff
@@ -0,0 +1,56 @@
+import styled from "@emotion/styled";
+import type { AnchorHTMLAttributes } from "react";
+
+import { ResponsiveChild } from "metabase/components/ResponsiveContainer/ResponsiveContainer";
+import { color } from "metabase/lib/colors";
+import type { AnchorProps } from "metabase/ui";
+import { Anchor } from "metabase/ui";
+
+import type { RefProp } from "./types";
+
+export const Breadcrumb = styled(Anchor)<
+  AnchorProps &
+    AnchorHTMLAttributes<HTMLAnchorElement> &
+    RefProp<HTMLAnchorElement>
+>`
```

Comment:
@npfitz I tried to make this `Breadcrumb` component into a `Link` and it took me down a time-consuming rabbit-hole. It's because I need to pass this component a ref and it turns out Links don't play well with refs (in this version of React Router). Would it be OK if we leave `Breadcrumb` as an `Anchor` for now? Perhaps we can revisit this performance tune-up after the release.